### ISSUE TITLE
Numeric value out of range: 1416 Cannot get geometry object from data you send to the GEOMETRY field

### ISSUE THREAD

**Issue by Edmonbelchev** [MatanYadaev/laravel-eloquent-spatial#104]
```
 public function setLocationAttribute($value) {
        $this->attributes['location'] = new Point($value['lng'], $value['lat']);
    }
```

When I try to update my location column trough setAttribute method I get the following error:
Numeric value out of range: 1416 Cannot get geometry object from data you send to the GEOMETRY field

If I update the column with raw sql query everything works fine.

Using Laravel 10.10 & mysql 8.0

**Comment by MatanYadaev** [comment]
Can you please fork and add a failing test?

**Comment by reefki** [comment]
I'm facing the same issue, here is my temporary solution that you could try

```
$location = new Point($value['lng'], $value['lat']);

public function setLocationAttribute($value) {
    $this->attributes['location'] = DB::raw("ST_GeomFromText('{$location}')");
}
```

**Comment by jaetoole** [comment]
Same issue here!

**Comment by MatanYadaev** [comment]
Guys, I need your help to reproduce this issue. Please add a failing test to the repository, or provide a complete code snippet.

**Comment by Edmonbelchev** [comment]
Sorry for the late response. I've managed to make it work but not by using the setAttribute method.
Instead of using the setAttribute method, I identified that it was causing a Numeric value out of range error (1416) and interfering with the GEOMETRY field.

For an alternative approach. Within my model's request class, I created a function called location():
```
public function location()
{
    // Extract latitude and longitude from the request (adjust accordingly)
    $latitude = $this->input('location.coordinates.0');
    $longitude = $this->input('location.coordinates.1');

    // Create a Point object with spatial coordinates
    return new Point($latitude, $longitude);
}
```
Then, in the controller's store method, I simply call this function to store the geometric value:
`['location' => $request->location()]`
